pkgs/top-level/python-packages.nix

```diff
@@ -32086,6 +32086,26 @@ EOF
 
   treq = callPackage ../development/python-modules/treq { };
 
+  snakeviz = buildPythonPackage rec {
```

Comment:
Done did it. 

Where is this policy documented? I want to see if I just missed this or if it should be moved somewhere more prominent in the nixpkgs manual.